microsoft-edge/devtools-guide-chromium/whats-new/2023/06/devtools-114.md

```diff
@@ -20,174 +20,162 @@ ms.date: 05/31/2023
+Also, a new **Style Sheet** column has been added to the **Selector Stats** tab.  The **Style Sheet** column contains links for each selector back to the stylesheet where the selectors are defined.
 
 ![Selector Stats are now aggregated across Recalculate Style events within the currently displayed section of the recorded profile](./devtools-114-images/aggregate-selector-stats.png)
 
-<!-- todo
-1. Open Devtools and navigate to the performance tool
-2. Hit the capture settings button
-3. Check the box at the top that says "Enable advanced rendering instrumentation (slow)"
-4. Click on a purple "Recalculate style" box
-5. Click on the Selector stats button on the bar below
--->
-
 Thank you for using the **Selector Stats** feature and sharing your feedback with us in [GitHub Issue #98: \[Feedback\] Selector Performance Tracing Explainer](https://github.com/MicrosoftEdge/DevTools/issues/98)!
 
 See also:
 * [Analyze selector performance during Recalculate Style events](../../../evaluate-performance/selector-stats.md)
 * [GitHub Issue #98: \[Feedback\] Selector Performance Tracing Explainer](https://github.com/MicrosoftEdge/DevTools/issues/98)
+* [Debug long-running Recalculate Style events with new selector stats](../01/devtools-109.md#debug-long-running-recalculate-style-events-with-new-selector-stats) in _What's new in DevTools (Microsoft Edge 109)_.
+
 
 <!-- ====================================================================== -->
-## The Issues tool warns you when CSS properties trigger reflow 
+## The Issues tool and Styles pane warn about CSS properties that trigger reflow
 
-<!-- Subtitle: Reflow is when the browser re-calculates the positions and geometries of elements in the page and is user-blocking so limit reflow as much as possible for better performance.-->
+<!-- Subtitle: The Styles pane shows a wavy underline on CSS properties that trigger reflow, and the Issues tool warns about such properties. This notification helps limit re-calculating positions and geometries of elements, to avoid blocking user interaction. -->
 
 <!-- Reviewer: Vidal Guillermo Diazleal Ortega -->
 
-Reflow is the name of the web browser process for re-calculating the positions and geometries of elements in the document, for the purpose of re-rendering part or all of the document. Because reflow is a user-blocking operation in the browser, from a performance perspective, we recommend limiting reflow as much as possible to keep your web content smooth and interactable. Some CSS properties don't trigger reflow because they run on the compositor thread in the browser, like [transform](https://developer.mozilla.org/docs/Web/CSS/transform) and [opacity](https://developer.mozilla.org/docs/Web/CSS/opacity).
+DevTools now detects and warns about properties that can cause performance issues when animated.
+*  In the **Styles** pane in the **Elements** tool, a wavy underline and tooltip has been added on CSS properties that trigger reflow.
```

Comment:
I suggest merging this content with the sentence before the image.

Reply:
done locally